.circleci/config.yml

```diff
@@ -37,8 +37,13 @@ jobs:
             docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
             docker buildx create --name multiarch --driver docker-container --use
             docker buildx inspect --bootstrap
+            CORE_VERSION_WITH_PATCH=$(cat Dockerfile | grep "ARG CORE_VERSION=" | cut -d'=' -f2)
             CORE_VERSION=$(cat Dockerfile | grep "ARG CORE_VERSION=" | cut -d'=' -f2 | cut -d'.' -f1,2)
-            docker buildx build --platform $BUILDX_PLATFORMS -t supertokens/supertokens-postgresql:${CORE_VERSION} -o type=image,push=true .
+            if [ "$CORE_VERSION" != "$CORE_VERSION_WITH_PATCH" ]; then
+              docker buildx build --platform $BUILDX_PLATFORMS -t supertokens/supertokens-postgresql:${CORE_VERSION} -t supertokens/supertokens-postgresql:${CORE_VERSION_WITH_PATCH} -o type=image,push=true .
```

Comment:
Have you tested that all the tags will get pushed to docker hub while building the image this way? If not, you can create a docker hub account and try creating a random image with multiple tags as how you have done here and push. Verify that all the tags are present for the image.

Reply:
Tested, works fine.